modules/core/packet-server/types/merkle.go

```diff
@@ -5,6 +5,8 @@ import (
 	commitmenttypesv2 "github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
 )
 
+const SubModuleName = "packetserver"
```

Comment:
We could add a `keys.go` file in `types` and move this there. I think that would be a better place for now.